Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1916,28 +1945,16 @@ extension StatusTableViewController: BluetoothStateManagerObserver {
     }
 }
 
-private class DelegateShim: CGMManagerSetupViewControllerDelegate {
```

Comment:
[nit] IMO, I don't like this change.  I think using a `Delegate` makes the coded harder to follow in many cases, especially if there's a specific startup sequence that needs to be followed.  Please consider putting this back.

Reply:
There was a bug here; the Minimed cgm setup wasn't actually working. And I found the previous code quite confusing to follow. "DelegateShim" is a really vague name for a type, and I still think `maybeSetupCGMManager` is a poor function name. Is it going to setup the CGMManager or not? But I didn't tackle that here.